geopandas/_vectorized.py

```diff
@@ -638,6 +638,18 @@ def exterior(data):
         return _unary_geo("exterior", data)
 
 
+def offset_curve(data, distance, quad_segs=8, join_style="round", mitre_limit=5.0):
+    if compat.USE_SHAPELY_20:
+        return shapely.offset_curve(data, distance, quad_segs, join_style, mitre_limit)
```

Comment:
```suggestion
        return shapely.offset_curve(data, distance=distance, quad_segs=quad_segs, join_style=join_style, mitre_limit=mitre_limit)
```

Can we spell it out to be more future-proof in case of some API changes on the shapely side?